.env

```diff
@@ -1,3 +0,0 @@
-GITHUB_CLIENT_ID=740549102466a19f21b1
-GITHUB_CLIENT_SECRET=a2d032ab1bb3f5edc722b037a689cd8bc98afa3d
-GITHUB_API_LOGOUT_URL=https://api.github.com/applications/740549102466a19f21b1/grant
```

Comment:
This is still visible 👀

Reply:
Har to remove when pushed to github. These values are at the moment an test account and not the ones we will be using when deploying. Since we haven't merged this branch into main yet there is a possibility to get rid of this by doing a interactive rebase where the commit adding this 3 lines can be removed and then force push the branch.

By just doing a delete in a new commit will still have the original commit for eternity with us. Another possible way to solve the problem might be to use the option merge squashing that will just do one commit.